packages/functional-tests/pages/products/index.ts

```diff
@@ -44,10 +46,30 @@ export class SubscribePage extends BaseLayout {
     await paypalWindow.click('button[id=consentButton]');
   }
 
+  async addExpiredCoupon() {
+    await this.page.click('[data-testid="coupon-input"]');
+    await this.page.fill('[data-testid="coupon-input"]', 'autoexpired');
+    await this.page.click('[data-testid="coupon-button"]');
```

Comment:
nit: for a future PR, but we should probably DRY this up a bit and move the selectors into some locator or something up in the POM so we aren't repeating IDs (in the slight chance we remove `data-testid`s in the future and refactor to something else).